PWGHF/D2H/TableProducer/dataCreatorDplusPiReduced.cxx

```diff
@@ -211,241 +215,231 @@ struct HfDataCreatorDplusPiReduced {
     return true;
   }
 
-  template <bool doMc = false, typename P, typename T>
-  void runDataCreation(aod::Collisions const& collisions,
-                       CandsDFiltered const& candsD,
+  template <bool doMc = false, bool withMl = false, typename P, typename T, typename C>
```

Comment:
super minor point: maybe you can remove the default values for `doMC` and `withMl`